.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
```suggestion
    "source.fixAll.eslint": true
```

Reply:
"explict" is desired (though we should stick that in a separate PR), as `source.fixAll` now expect a string rather than a boolean value, and this changing is VSCode updating itself to use the new value: https://code.visualstudio.com/updates/v1_83#_code-actions-on-save-and-auto-save